Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+precommit: FORCE tidy lint csv
+
+csv: FORCE
+	go run . csv github.com/google/go-licenses
```

Comment:
Could we make this a go test rather than a makefile command? 🤔

Reply:
Ideally I want a full e2e test for go-licenses CLI, when I searched on google, there are multiple approaches and it's a bit complicated. Do you have a solution you would recommend?
I'd split this out of this PR to control PR size.

Reply:
Put up another PR for this: https://github.com/google/go-licenses/pull/102

Reply:
Splitting it out SGTM. Thanks!